src/diffusers/pipelines/pixart_alpha/pipeline_pixart_alpha.py

```diff
@@ -688,6 +689,7 @@ def __call__(
         callback_steps: int = 1,
         clean_caption: bool = True,
         use_resolution_binning: bool = True,
+        model_token_max_length: int = 120,
```

Comment:
should we maybe make this rather part of the pipeline config (`model_index.json`) ? Wdyt @sayakpaul